pyproject.toml

```diff
@@ -127,6 +127,7 @@ oauth2client = "4.1.3"
 regex = ">=2020.6,<2020.10"
 joblib = "^0.15.1"
 sentry-sdk = "^0.17.4"
+colorama = {version = "^0.4.4", platform = "windows"}
```

Comment:
Should we go for `sys_platform == 'win32'"` as this option is official documented?